src/pages/PassportRedemption/SignIn/uploadScreen.tsx

```diff
@@ -118,14 +128,16 @@ const UploadScreen = () => {
             onChange={(e) => {
```

Comment:
i would make this a separate function for readability and also for the reason down below in the next comment